packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -478,6 +488,10 @@ export function makeRumPublicApi(
 
     startView,
 
+    updateViewName: (name) => {
+      strategy.updateViewName(name)
+    },
```

Comment:
**💬 suggestion:** ‏don't expose the method the experimental feature is not enabled, similarly to `startDurationVital` above. Don't expose the type/documentation either.